### DISCUSSION TITLE
Two questions: arrows and zoom

### DISCUSSION THREAD

**Discussion by garulovilla** [pubpub-zz/ppInk#266] [Q&A]
Hi, this is a very useful tool to presentations, thanks for all the effort.

I have two questions:

1. How to get the simple version of arrows? I mean, when I draw an arrow, start like "--->" but when I release the left mouse button, the arrowhead is converted to the customized version (where do you can load the images for defining the tail and the head). Personally, I like more the version of this [image](https://github.com/pubpub-zz/ppInk/blob/master/penDialog.png). The only option that I found is create an image with the form that like, but I wonder if exist a configuration to return or take the original arrow.

2. And referent with the zoom, it's possible to only have the magnifier option? The magnifier is shown when I use the Z key, but when I type again the Z for "quit" the zoom, it passes to another option (spot on cursor) and then another Z to quit the zoom.

**Comment by pubpub-zz** [comment]
> Hi, this is a very useful tool to presentations, thanks for all the effort.

appreciated

> 
> I have two questions:
> 
>     1. How to get the simple version of arrows? I mean, when I draw an arrow, start like "--->" but when I release the left mouse button, the arrowhead is converted to the customized version (where do you can load the images for defining the tail and the head). Personally, I like more the version of this [image](https://github.com/pubpub-zz/ppInk/blob/master/penDialog.png). The only option that I found is create an image with the form that like, but I wonder if exist a configuration to return or take the original arrow.

Unfortunately the approach is totally different and I've moved to new design where ends are bitmaps. However you can build you own bitmap to match the old visual.

> 
>     2. And referent with the zoom, it's possible to only have the magnifier option? The magnifier is shown when I use the Z key, but when I type again the Z for "quit" the zoom, it passes to another option (spot on cursor) and then another Z to quit the zoom.

if you should be able to select the zoom you are looking in the options:
![image](https://github.com/pubpub-zz/ppInk/assets/4083478/74451018-10b3-4b3c-b76c-6d49fe96204a)

with 1 you should get what you are looking for

**Reply by garulovilla** [reply]
Thanks for the quick response.

1. OK, I understand, thanks for the info.
2. That's curious, I actually have that option activated, but when I press de Z key pass to the other option (lamp).

![](https://github.com/pubpub-zz/ppInk/assets/4468146/b35d9b14-f81c-45cc-8d90-936b7831821b)
